tests/FINDOLOGIC/Export/Tests/XmlSerializationTest.php

```diff
@@ -409,7 +409,7 @@ public function unsupportedValueProvider()
      * @param string $method Name of the method to call to interact with an unsupported value.
      * @param mixed $parameter The parameter in case of a setter.
      */
-    public function testUsingValuesUnsupportedByXmlCauseExceptions($method, $parameter)
+    public function testUsingValuesUnsupportedByXmlCauseExceptions(string $method, $parameter)
```

Comment:
Type of `$parameter` is missing, which can be `?float`.

Reply:
Fixed